Code/Framework/AzCore/Tests/Math/Vector4Tests.cpp

```diff
@@ -102,44 +103,46 @@ namespace UnitTest
 
         // operation ( r.x = (cmp1.x > cmp2.x) ? vA.x : vB.x ) per component
         Vector4 compareGreaterAB = Vector4::CreateSelectCmpGreater(vA, vB, Vector4(1.0f), Vector4(0.0f));
-        AZ_TEST_ASSERT(compareGreaterAB.IsClose(Vector4(0.0f, 0.0f, 1.0f, 0.0f)));
+        EXPECT_THAT(compareGreaterAB, IsClose(Vector4(0.0f, 0.0f, 1.0f, 0.0f)));
         Vector4 compareGreaterCA = Vector4::CreateSelectCmpGreater(vC, vA, Vector4(1.0f), Vector4(0.0f));
-        AZ_TEST_ASSERT(compareGreaterCA.IsClose(Vector4(1.0f, 1.0f, 0.0f, 0.0f)));
+        EXPECT_THAT(compareGreaterCA, IsClose(Vector4(1.0f, 1.0f, 0.0f, 0.0f)));
     }
 
     TEST(MATH_Vector4, TestGetSet)
     {
+        const float values[4] = { 10.0f, 20.0f, 30.0f, 40.0f };
```

Comment:
unused?

Reply:
moved it down to the line where its being used.